cmd/theme/main.go

```diff
@@ -146,97 +146,77 @@ func main() {
 	output.Flush()
 }
 
-func NoOpParser(cmd string, args []string) (result map[string]interface{}, set *flag.FlagSet) {
-	return make(map[string]interface{}), nil
+func NoOpParser(cmd string, args []string) (result commands.Args, set *flag.FlagSet) {
+	return commands.DefaultArgs(), nil
 }
 
-func FileManipulationCommandParser(cmd string, args []string) (result map[string]interface{}, set *flag.FlagSet) {
-	result = make(map[string]interface{})
+func FileManipulationCommandParser(cmd string, args []string) (result commands.Args, set *flag.FlagSet) {
+	result = commands.DefaultArgs()
 	currentDir, _ := os.Getwd()
-	var environment, directory string
 
 	set = makeFlagSet(cmd)
-	set.StringVar(&environment, "env", themekit.DefaultEnvironment, "environment to run command")
-	set.StringVar(&directory, "dir", currentDir, "directory that config.yml is located")
+	set.StringVar(&result.Environment, "env", themekit.DefaultEnvironment, "environment to run command")
+	set.StringVar(&result.Directory, "dir", currentDir, "directory that config.yml is located")
 	set.Parse(args)
 
-	result["themeClient"] = loadThemeClient(directory, environment)
-	result["filenames"] = args[len(args)-set.NArg():]
+	result.ThemeClient = loadThemeClient(result.Directory, result.Environment)
+	result.Filenames = args[len(args)-set.NArg():]
 	return
 }
 
-func WatchCommandParser(cmd string, args []string) (result map[string]interface{}, set *flag.FlagSet) {
-	result = make(map[string]interface{})
+func WatchCommandParser(cmd string, args []string) (result commands.Args, set *flag.FlagSet) {
+	result = commands.DefaultArgs()
 	currentDir, _ := os.Getwd()
 	var allEnvironments bool
-	var environment, directory, notifyFile string
-	var environments themekit.Environments
 
 	set = makeFlagSet(cmd)
-	set.StringVar(&environment, "env", themekit.DefaultEnvironment, "environment to run command")
+	set.StringVar(&result.Environment, "env", themekit.DefaultEnvironment, "environment to run command")
 	set.BoolVar(&allEnvironments, "allenvs", false, "start watchers for all environments")
```

Comment:
@ilikeorangutans Yes, it is: https://github.com/Shopify/themekit/pull/135/files#diff-1480839ab7d21ce62f762376dde0c0a0R181